package-lock.json

```diff
@@ -1,35 +1,7 @@
 {
   "name": "handbook",
-  "version": "0.0.1",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
-    "": {
-      "name": "handbook",
-      "devDependencies": {
-        "prettier": "^2.6.1"
-      },
-      "engines": {
-        "node": ">=16.14.2",
-        "npm": "^8.5.0"
-      }
-    },
-    "node_modules/prettier": {
-      "version": "2.6.1",
-      "resolved": "https://registry.npmjs.org/prettier/-/prettier-2.6.1.tgz",
-      "integrity": "sha512-8UVbTBYGwN37Bs9LERmxCPjdvPxlEowx2urIL6urHzdb3SDq4B/Z6xLFCblrSnE4iKWcS6ziJ3aOYrc1kz/E2A==",
-      "dev": true,
-      "bin": {
-        "prettier": "bin-prettier.js"
-      },
-      "engines": {
-        "node": ">=10.13.0"
-      },
-      "funding": {
-        "url": "https://github.com/prettier/prettier?sponsor=1"
-      }
-    }
-  },
+  "lockfileVersion": 1,
```

Comment:
**issue:** We should use the latest lockfile version `2`. Could we make sure we use the correct version of node and npm, that is `node >=16.14.2` and `npm ^8.5.0`? To check , run

```sh
node --version
npm --version
```

Reply:
https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl

This was very helpful to update my versions. However, we shall see how successful it is